lib/team_statistics.rb

```diff
@@ -80,8 +80,14 @@ def head_to_head(team_id)
     def team_name(team_id)
         @teams.find { |team| team.team_id == team_id }.team_name
     end
-        
-   <<<<<<< oscar-team-stat-m
+  
+    def favorite_opponent(team_id)
+       head_to_head_results = head_to_head(team_id)
```

Comment:
glad you were able to build off seths method. Him and I discussed if it would work for mine but unfortunately didnt work out. good job!